frontend/src/app/side-menu/side-menu.component.ts

```diff
@@ -60,10 +70,18 @@ export class SideMenuComponent {
     console.log(file)
 
     this.sharedService.sendFormData(formData).subscribe((event: any) => {
-      if (typeof (event) === 'object')  {
-
-        if (event.body != undefined){
-          this.sharedService.nextMessage(event.body)
+      if (typeof (event) === 'object') {
```

Comment:
for function calls like this typeof, no space before (. This line should be:
      if (typeof(event) === 'object') {